### PR TITLE
Added 1 unit test and dependencies required for running the test

### PR DESCRIPTION
**Pull request by sherwoodf**
- Added a single unit test. It currently relies on the example data not having various fields to not send requests to the api. It would be better to mock this, but this was getting complicated & made me want to make changes to the cli code, so thought i would stop there for this PR
- Added vscode folder and re-formatted files according to linter.

### CODE REVIEW

File: tests/data/input/bia_image.json
Comment (kbab) [inline tests/data/input/bia_image.json:1]: There has been a change in the name of `image_acquisition_methods_uuid` in `bia_integrator` that is causing an issue when I try running the tests locally...

File: tests/test_unit_model_transform.py
Comment (kbab) [inline tests/test_unit_model_transform.py:1]: Should `mocker` on line 14 be commented out?

File: tests/test_unit_model_transform.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from unittest.mock import patch, Mock
+
+
+def test_bia_image_to_export_image(
+    bia_image: api_models.BIAImage,
+    bia_study: api_models.BIAStudy,
+    website_image: ExportImage,
+    mocker,
+):
+    # mocker.patch("rw_client.get_image_acquisition").return_value.specimen_uuid = (
```
Comment (sherwoodf) [inline tests/test_unit_model_transform.py]: Shouldn't be in this - will remove

File: tests/test_unit_model_transform.py
Comment (sherwoodf) [reply tests/test_unit_model_transform.py:1]: Failed to save some changes before committing. Fixed.

File: tests/data/input/bia_image.json
Comment (sherwoodf) [reply tests/data/input/bia_image.json:1]: Yep, i need to update the whole package, but would rather do this in a separate PR

**Review by AybukeKY — APPROVED** [review]

**Review by kbab — APPROVED** [review]
LGTM

**Pull request merged**